packages/tests/src/commonlib/containeraAppValidator.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureScopes } from "@microsoft/teamsfx-core/build/common/tools";
+import * as chai from "chai";
+
+import MockAzureAccountProvider from "@microsoft/teamsapp-cli/src/commonlib/azureLoginUserPassword";
+import { EnvConstants } from "./constants";
+
+import {
+  getContainerAppProperties,
+  getSubscriptionIdFromResourceId,
+  getResourceGroupNameFromResourceId,
+} from "./utilities";
+import { Executor } from "../utils/executor";
+import { Env } from "../utils/env";
+
+export class ContainerAppValidator {
+  private ctx: any;
+  private subscriptionId: string;
+  private rg: string;
+  private containerAppName: string;
+
+  constructor(ctx: any) {
+    console.log("Start to init validator for Azure Container App.");
+
+    this.ctx = ctx;
+
+    const resourceId = ctx[EnvConstants.AZURE_CONTAINER_APP_RESOURCE_ID];
+    chai.assert.exists(resourceId);
+    this.subscriptionId = getSubscriptionIdFromResourceId(resourceId);
```

Comment:
I think we can get subscription id from ctx directly. Is it necessary to parse the resourceId?

Reply:
I've tried to retrieve the resourceId directly from ctx, but I wasn't able to find it. The content included in ctx is as shown in the screenshot.
![image](https://github.com/OfficeDev/TeamsFx/assets/139844715/513aca95-0ced-496a-9e05-8b1d944cdaaf)